src/NumericalAlgorithms/Spectral/Mesh.hpp

```diff
@@ -199,8 +233,8 @@ class Mesh {
    * \see slice_through() The templated overload of this function
    */
   template <size_t SliceDim, Requires<(SliceDim <= Dim)> = nullptr>
-  Mesh<SliceDim> slice_through(const std::array<size_t, SliceDim>& dims) const
-      noexcept;
+  Mesh<SliceDim> slice_through(
+      const std::array<size_t, SliceDim>& dims) const noexcept;
```

Comment:
My clang-format also suggests this formatting, but `git-clang-format` doesn't. I think one of them reads the `.clang-format` file in the repo root and the other doesn't (maybe). That's as far as I have looked into this, but it sure annoys me occasionally.

I suggest you run `git-clang-format develop` on this branch. Formatting-only changes are annoying to review because they obscure real changes. If want to fix existing formatting-issues in a file I recommend you interactively rebase to jump ahead of your commits, run clang-format over the file and make a separate commit. Such formatting-only commits are super easy to review and don't distract from the real diff :)

If you do this, please force-push the formatting edits directly instead of pushing them as fixup commits to keep the git history in the PR manageable.